plugin-catalog/src/components/plugins/InstalledList.tsx

```diff
@@ -17,53 +17,124 @@ interface Plugin {
-              getter: plugin => (
-                <Box>
-                  <Link
-                    routeName="/plugin-catalog/:repoName/:pluginName"
-                    params={{ repoName: plugin.repoName, pluginName: plugin.folderName }}
-                  >
-                    {plugin.pluginTitle}
-                  </Link>
-                </Box>
-              ),
-            },
-            {
-              label: 'Version',
-              getter: plugin => plugin.pluginVersion,
-            },
-            {
-              label: 'Repo',
-              getter: plugin => plugin.repoName,
-            },
-            {
-              label: 'Author',
-              getter: plugin => plugin.author,
-            },
-          ]}
-          emptyMessage="No plugins installed"
-          data={installedPlugins || []}
-        />
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 2,
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 2,
+              alignItems: 'start',
+            }}
+          >
+            <Typography variant="h6" components="h2">
+              Installed from the Plugin Catalog
+            </Typography>
+            <SimpleTable
+              columns={[
+                {
+                  label: 'Name',
+                  getter: plugin => (
+                    <Box>
+                      <Link
+                        routeName="/plugin-catalog/:repoName/:pluginName"
+                        params={{ repoName: plugin.repoName, pluginName: plugin.folderName }}
+                      >
+                        {plugin.pluginTitle}
+                      </Link>
+                    </Box>
+                  ),
+                },
+                {
+                  label: 'Version',
+                  getter: plugin => plugin.pluginVersion,
+                },
+                {
+                  label: 'Repo',
+                  getter: plugin => plugin.repoName,
+                },
+                {
+                  label: 'Author',
+                  getter: plugin => plugin.author,
+                },
+              ]}
+              emptyMessage="No plugins installed"
+              data={installedPlugins || []}
+            />
+          </Box>
+
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 2,
+              alignItems: 'start',
+            }}
+          >
+            <Typography variant="h6" components="h2">
+              Other Installed Plugins
+            </Typography>
+
+            <SimpleTable
+              columns={[
+                {
+                  label: 'Name',
+                  getter: otherInstalledPlugins => (
+                    <Box>
+                      <Link
+                        routeName={`pluginDetails`}
```

Comment:
This lliteral is not replacing any var, so it may be `"pluginDetails"` instead (no `{}` needed either).